module/edge_logic.lua

```diff
@@ -497,7 +497,7 @@ function edge_transports.set_active_edges(json)
 
 		if not edge.active then
 			if edge.linked_belts then
-				for offset, link in pairs(edge.linked_belts) do
+				for _, link in pairs(edge.linked_belts) do
```

Comment:
Does replacing the name of the key here with an underscore make the code more readable? I get that offset is unused, but by calling it offset it's understandable what the key is, wheras now I have to go look at other code dealing with linked_belts to figure out what the key is.